lib/shortener.rb

```diff
@@ -5,18 +5,25 @@ module Shortener
   autoload :ActiveRecordExtension, "shortener/active_record_extension"
   autoload :ShortenUrlInterceptor, "shortener/shorten_url_interceptor"
 
+  AMBIGUOUS_CHARACTERS = ['i', 'o', 'l', 'I', 'O', '0', '1'].freeze
```

Comment:
Since ambiguous characters are somewhat font dependent and somewhat subjective, I'm very open to suggestions on what characters belong here. There doesn't seem to be strict consensus on the internet.